process/views.py

```diff
@@ -232,6 +232,21 @@ def metadata(self, request, pk=None):
 
         return Response(metadata)
 
+    @action(detail=True)
```

Comment:
The other views use `@extend_schema` to define the output format. This is only to improve the OpenAPI docs. You can create a follow-up issue instead.